repl/Main.hs

```diff
@@ -124,26 +124,27 @@ repl ctx startWorld = do
   forever $
     do putStr "> "
        l <- T.getLine
-       tryCommand theWorld l $ \l' -> case readExpr "<repl>" l' of
```

Comment:
Hmm, the diff algorithm is bad here... It appears that the only change is that you put this `case` on a new line, is that right?

Reply:
Thinking back, I remember being confused by having so much there and moving it for the sake of understanding it. So yes :)